spec/system/occupation_standards/index_spec.rb

```diff
@@ -149,6 +149,61 @@
     expect(page).to_not have_link "HR"
   end
 
+  it "can clear form", :js do
+    wa = create(:state, name: "Washington")
+    ra = create(:registration_agency, state: wa)
+    mechanic = create(:occupation_standard, :hybrid, :with_data_import, title: "Mechanic", onet_code: "12.3456", registration_agency: ra)
+    ma = create(:occupation_standard, :program_standard, :time, :with_data_import, title: "Medical Assistant", onet_code: "12.34567", registration_agency: ra)
```

Comment:
Does `ma` stand for `medical assistant`? If so, can we prefer using the entire word so newcomers to this codebase can understand it more easily?